cmd/controller/main.go

```diff
@@ -94,7 +97,15 @@ func main() {
 	if err != nil {
 		logger.Fatalf("fail to create kubernetes client, error: %v", err)
 	}
-	kubeClient := k8s.NewKubeClient(k8SClient, logger, objects.NewObjectLogger(), *namespace)
+	kubeClient := k8s.NewKubeClient(k8SClient, logger, objects.NewObjectLogger(), *namespace,
+		backoff.NewExponentialHandler(&grpcbackoff.Config{
+			// TODO(n.mikhnenko): customize vars
```

Comment:
TODO must be denoted as gh issue with link in the comment.

Reply:
used base config for this